deepchem/models/wandblogger.py

```diff
@@ -83,6 +83,9 @@ def __init__(self,
     self.wandb_init_params.update(**kwargs)
     self.initialized = False
 
+    # Dataset ids are used to differentiate datasets seen by the logger
+    self.dataset_ids = []  # type: List[Union[int, str]]
```

Comment:
It would be better to use a type annotation rather than a comment:

```python
self.dataset_ids: List[Union[int, str]] = []
```

Reply:
Will fix this!